controllers/virtualmachine/volumebatch/volumebatch_controller.go

```diff
@@ -326,92 +364,83 @@ func (r *Reconciler) getBatchAttachmentForVM(
 	return attachment, nil
 }
 
-func (r *Reconciler) processBatchAttachment(
+// processBatchAttachmentAndReturnVolumeStatus processes the batch attachment and returns the
+// volume statuses that are tracked by the batch attachment.
+func (r *Reconciler) processBatchAttachmentAndReturnVolumeStatus(
```

Comment:
can this be broken down into two functions? One for just returning the volume status?